src/style.css

```diff
@@ -1,3 +1,69 @@
 body {
   margin: 0;
 }
+
+html {
+  font-family: Roboto, serif;
```

Comment:
The font-family declaration should include fallback fonts in quotes if they contain spaces. Consider using 'Roboto', sans-serif; to ensure proper font rendering.